### PR TITLE
[ruby] the ruby extension repo contains two extensions

### PR DESCRIPTION
**Pull request by svenefftinge**

### CODE REVIEW

File: publish-extensions.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const readFile = util.promisify(fs.readFile);
       if (extension.checkout) {
         await exec(`git checkout ${extension.checkout}`, { cwd: '/tmp/repository' });
       }
-      const location = path.join('/tmp/repository', extension.location || '.');
+      const location = path.join('/tmp/repository', extension.installDir || extension.location || '.');
```
Comment (jankeromnes) [inline publish-extensions.js:62]: I don't see the point of `installDir` when there is already `location`. Please remove it.

File: extensions.json [hunk 2]
```diff
@@ -338,8 +338,17 @@
       "version": "0.27.0",
       "repository": "https://github.com/rubyide/vscode-ruby",
       "checkout": "v0.27.0",
+      "installDir": ".",
```
Comment (jankeromnes) [inline extensions.json:341]:
```suggestion
```

File: extensions.json [hunk 3]
```diff
@@ -338,8 +338,17 @@
...
       "location": "packages/vscode-ruby-client"
     },
+    {
+      "id": "wingrunr21.vscode-ruby",
+      "version": "0.27.0",
+      "repository": "https://github.com/rubyide/vscode-ruby",
+      "checkout": "v0.27.0",
+      "installDir": ".",
```
Comment (jankeromnes) [inline extensions.json:349]:
```suggestion
```

File: publish-extensions.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ const exec = require('./lib/exec');
 const readFile = util.promisify(fs.readFile);
 
 (async () => {
-  /** @type {{ extensions: { id: string, version?: string, repository: string, checkout?: string, location?: string }[] }} */
+  /** @type {{ extensions: { id: string, version?: string, repository: string, checkout?: string, location?: string, installDir?: string }[] }} */
```
Comment (jankeromnes) [inline publish-extensions.js:21]:
```suggestion
  /** @type {{ extensions: { id: string, version?: string, repository: string, checkout?: string, location?: string }[] }} */
```

File: publish-extensions.js (hunk 1 above)
Comment (jankeromnes) [inline publish-extensions.js:62]:
```suggestion
      const location = path.join('/tmp/repository', extension.location || '.');
```

**Comment by svenefftinge** [comment]
installDir != location for this, which is the whole point

File: publish-extensions.js [hunk 5]
```diff
@@ -59,11 +59,10 @@ const readFile = util.promisify(fs.readFile);
       if (extension.checkout) {
         await exec(`git checkout ${extension.checkout}`, { cwd: '/tmp/repository' });
       }
-      const location = path.join('/tmp/repository', extension.location || '.');
       let yarn = await new Promise(resolve => {
         fs.access(path.join(location, 'yarn.lock'), error => resolve(!error));
```
Comment (jankeromnes) [inline publish-extensions.js:63]:
```suggestion
        fs.access(path.join('/tmp/repository', 'yarn.lock'), error => resolve(!error));
```

**Pull request merged**